ui/js/constants.js

```diff
@@ -223,15 +223,14 @@ export const thFavicons = {
   unavailable: treeFavicon,
 };
 
-export const thRepoGroupOrder = {
-  development: 1,
-  'release-stabilization': 2,
-  'project repositories': 3,
-  'comm-repositories': 4,
-  'qa automation tests': 5,
-  try: 6,
-  taskcluster: 7,
-};
+export const thRepoGroupOrder = [
+  'development',
+  'release-stabilization',
+  'project repositories',
+  'comm-repositories',
+  'qa automation tests',
+  'other',
```

Comment:
The new `ci-admin` group (from #3770) needs adding here

Reply:
Ahh, nice catch!  Thanks!